lambda_src/drivers/process_https.py

```diff
@@ -44,14 +44,20 @@ def process_row(
             raise RuntimeError('url must start with https://')
     else:
         req_host = base_url
-        req_path = url or '/'
```

Comment:
not a blocker to merge, but I think we either missed or mis-merged some changes w.r.t. how `base_url` and `url` work together -- I think some unit tests would help here.

Reply:
@sfc-gh-afedorov Good catch raised another PR for it.